packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -1,4 +1,6 @@
 const { Command } = require('@oclif/command');
+const { stdtermwidth } = require('@oclif/plugin-help/lib/screen');
```

Comment:
This should be fine, but my understanding is that if they move or change their file structure at all, this import would break. Just a thing to be wary of.

Reply:
If the import breaks, the smoke test will break too, so I'm not too worried.